creator-node/default-config.json

```diff
@@ -15,7 +15,7 @@
   "rateLimitingMetadataReqLimit": 3000,
   "rateLimitingImageReqLimit": 6000,
   "rateLimitingTrackReqLimit": 6000,
-  "maxAudioFileSizeBytes":250000000,
+  "maxAudioFileSizeBytes":1000000000,
```

Comment:
Lets hold off on bumping up the audio file sizes until we can remove masters for newly uploaded tracks

Reply:
chatted offline. going to table til next week